CONTRIBUTORS.txt

```diff
@@ -271,4 +271,8 @@
   Maintenance of the build and test infrastructure.
 
 - Taylor D. Scott
-  Simplified _upsampled_dft and extended register_translation to nD images.
\ No newline at end of file
+  Simplified _upsampled_dft and extended register_translation to nD images.
+
+- Leonid Bloch
+  Implement non-local means restoration.
```

Comment:
I thought NL-means was implemented by @emmanuelle? And git blame corroborates my memory...

Reply:
Yes, Emma contributed the non-local means function and myself and others have mode some improvements over the years.

I would describe the contribution here as "Improvements to the performance and stability of non-local means restoration."